src/main/java/com/flightstats/hub/app/ClusterHubBindings.java

```diff
@@ -35,8 +38,11 @@ protected void configure() {
                 .to(ClusterContentService.class).asEagerSingleton();
         bind(RemoteSpokeStore.class).asEagerSingleton();
         bind(ContentDao.class)
```

Comment:
Can the two ContentDao bindings here and in SingleHubBindings be moved up to HubBindings?

Reply:
Yeah. In fact I had that in place but a couple rollbacks I did must have dropped the change. I'll put it back.